helm/polaris/values.yaml

```diff
@@ -0,0 +1,391 @@
+  successThreshold: 1
+  # -- Minimum consecutive failures for the probe to be considered failed after having succeeded. Minimum value is 1.
+  failureThreshold: 3
+  # -- Number of seconds after which the probe times out. Minimum value is 1.
+  timeoutSeconds: 10
+  # -- Optional duration in seconds the pod needs to terminate gracefully upon probe failure. Minimum value is 1.
+  terminationGracePeriodSeconds: 30
+
+# -- Configures the readiness probe for polaris pods.
+readinessProbe:
+  # -- Number of seconds after the container has started before readiness probes are initiated. Minimum value is 0.
+  initialDelaySeconds: 5
+  # -- How often (in seconds) to perform the probe. Minimum value is 1.
+  periodSeconds: 10
+  # -- Minimum consecutive successes for the probe to be considered successful after having failed. Minimum value is 1.
+  successThreshold: 1
+  # -- Minimum consecutive failures for the probe to be considered failed after having succeeded. Minimum value is 1.
+  failureThreshold: 3
+  # -- Number of seconds after which the probe times out. Minimum value is 1.
+  timeoutSeconds: 10
+
+# -- Advanced configuration via Environment Variables.
+# Extra environment variables to add to the Polaris server container.
+# You can pass here any valid EnvVar object:
+# https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.27/#envvar-v1-core
+# This can be useful to get configuration values from Kubernetes secrets or config maps.
+extraEnv:
+  []
+#  - name: AWS_STORAGE_BUCKET
+#    value: s3://xxxxx/
+#  - name: AWS_ACCESS_KEY_ID
+#    valueFrom:
+#      secretKeyRef:
+#        name: aws-secret
+#        key: access_key_id
+#  - name: AWS_SECRET_ACCESS_KEY
+#    valueFrom:
+#      secretKeyRef:
+#        name: aws-secret
+#        key: secret_access_key
+
+# -- Configures whether to enable the bootstrap metastore manager job
+bootstrapMetastoreManager: false
+
+# -- The secret name to pull persistence.xml from (ensure the key name is 'persistence.xml')
+persistenceConfigSecret: ~
+
+# -- Configures for polaris-server.yml
+polarisServerConfig:
+  server:
+    # Maximum number of threads.
+    maxThreads: 200
+
+    # Minimum number of thread to keep alive.
+    minThreads: 10
+    applicationConnectors:
+      # HTTP-specific options.
+      - type: http
+
+        # The port on which the HTTP server listens for service requests.
+        port: 8181
+
+    adminConnectors:
+      - type: http
+        port: 8182
+
+    # The hostname of the interface to which the HTTP server socket wil be found. If omitted, the
+    # socket will listen on all interfaces.
+    #bindHost: localhost
+
+    #  ssl:
+    #    keyStore: ./example.keystore
+    #    keyStorePassword: example
+    #
+    #    keyStoreType: JKS # (optional, JKS is default)
+
+    # HTTP request log settings
+    requestLog:
+      appenders:
+        # Settings for logging to stdout.
+        - type: console
+
+        # Settings for logging to a file.
+        - type: file
```

Comment:
Should file-based logging be enabled by default for K8s deployments? Isn't console enough?

Reply:
For this one, we won't need file-based logging for k8s. Should we remove it? Reasoning on putting it there is because that is the complete content of Polaris Server config (https://github.com/apache/polaris/blob/main/polaris-server.yml). I can remove the file appender if that is more preferred?

Reply:
I'm fine either way. You could also comment out the file section but leave it in here just in case, wdyt?